backend/src/index.d.ts

```diff
@@ -0,0 +1,8 @@
+import { type UserFindResponseDto } from './bundles/users/types/types.js';
+
+declare module 'fastify' {
+    // eslint-disable-next-line @typescript-eslint/consistent-type-definitions
+    interface FastifyRequest {
+        user: UserFindResponseDto;
```

Comment:
I think in out case it should be optional field, because in WHITE endpoints req.user = undefined
```suggestion
        user?: UserFindResponseDto;
```